src/user/user.controller.ts

```diff
@@ -13,8 +14,9 @@ export class UserController {
   }
 
   @Post('/login')
-  async login(@Body() body: LoginRequest): Promise<User> {
-    return this.service.findUserByUsernameAndPassword(body.username, body.password);
+  async login(@Body() body: LoginRequest): Promise<User> {    
```

Comment:
Tot asa, functia nu va returna cu User, ci un JSON care va avea ca membru tokenul ex: { token: <result from login> }. Pentru asta va trebui sa creezi o interfata in folderul de requests cu aceasta specificatie